src/renderer/components/dialog/FileImport.tsx

```diff
@@ -31,41 +31,46 @@ export class FileImport extends React.Component<FileImportProps, undefined> {
         const {__} = this.props;
         return (
             <div>
-                { this.buildBasicFileImportList() }
-                <div>
-                    <button className={ styles.primary } onClick={ this.importFiles }>{ __("dialog.yes") }</button>
-                    <button onClick={ this.props.closeDialog }>{ __("dialog.no") }</button>
-                </div>
+                { this.buildBasicFileImportList(__) }
+
             </div>
         );
     }
 
     private importFiles() {
-        const paths = this.props.files.map((file: File) => {
-            return file.path;
-        });
-        this.props.importFiles({ paths });
-        this.props.closeDialog();
+        if (this.props.files) {
+            const paths = this.props.files.map((file: File) => {
+                return file.path;
+            });
+            this.props.importFiles({ paths });
+            this.props.closeDialog();
+        }
     }
 
-    private buildBasicFileImportList() {
+    private buildBasicFileImportList(__: any) {
```

Comment:
No need to add the function parameter `__: any`, as you can see inside the function body there is already usage of `this.props.__`.